clrenv/lazy_env.py

```diff
@@ -206,6 +206,9 @@ def _apply_functions(d, recursive=False):
                     except EndpointConnectionError as e:
                         raise RuntimeError(
                             "clrenv could not connect to AWS to fetch parameters. If you're developing locally, try setting the offline environment variable (CLRENV_OFFLINE_DEV) to use placeholder values.")
+                    except _get_ssm_client().exceptions.ParameterNotFound as e:
```

Comment:
this seems like a strange way to get the exception type. Is there no way to import the error type without getting it off the client?

Reply:
Per here it seems like the canonical way to do it: https://stackoverflow.com/questions/46174385/properly-catch-boto3-errors

It can't be imported directly from `botocore`, strangely.